WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -37,7 +37,7 @@ public CoinJoinManager(WalletManager walletManager, RoundStateUpdater roundStatu
 	public ServiceConfiguration ServiceConfiguration { get; }
 	private CoinRefrigerator CoinRefrigerator { get; } = new();
 	public bool IsUserInSendWorkflow { get; set; }
-	private CancellationTokenSource AbortCancellationTokenSource { get; } = new ();
+	private CancellationTokenSource AbortCancellationTokenSource { get; } = new();
```

Comment:
There is a PR by David that improves this and makes this cts unnecessary. I would suggest you to remove it.

Reply:
I reverted the formatting changes.